LIMS_IMAGE/web/laboratory/models.py

```diff
@@ -26,6 +26,7 @@ def __str__(self):
         return self.name # tests are referenced by their name
     # By default, Django gives each model an auto-incrementing primary key with the type specified per app
     name = models.CharField(max_length=100) # name of the test
+    sample_type = models.CharField(max_length=100, null=True)
```

Comment:
please add a comment here explaining what this attribute is used for

Reply:
So I can go to the admin page and add those sample type and save it now from the sample-type branch. 
Then I can backup the database.

Reply:
yes

Reply:
Just to confirm, sample types are entered into the database in the admin page.  
![image](https://user-images.githubusercontent.com/62345938/152209980-b526f7d2-60dc-4c9a-abdf-12d2a8bcc9ee.png)

So now I can backup the database.